src/ansys/fluent/core/search.py

```diff
@@ -537,8 +537,11 @@ def _search_semantic(search_string: str, language: str, api_tree_data: dict):
                 ):
                     similar_keys.add(api_object_synset_name + "*")
     if similar_keys:
+        results = []
         for key in similar_keys:
-            return _search_wildcard(key, api_tree_data)
+            if _search_wildcard(key, api_tree_data):
```

Comment:
@hpohekar One could use a local variable to avoid two identical calls.

Reply:
We will update this now. Thank you.

Reply:
Done. Thank you.